files/build_templates/sonic_debian_extension.j2

```diff
@@ -538,6 +538,9 @@ sudo LANG=C cp $SCRIPTS_DIR/radv.sh $FILESYSTEM_ROOT/usr/local/bin/radv.sh
 # Copy sonic-netns-exec script
 sudo LANG=C cp $SCRIPTS_DIR/sonic-netns-exec $FILESYSTEM_ROOT/usr/bin/sonic-netns-exec
 
+# Copy sonic-sync-mode script
+sudo LANG=C cp $SCRIPTS_DIR/sonic-sync-mode $FILESYSTEM_ROOT/usr/bin/sonic-sync-mode
```

Comment:
>sonic-sync-mode [](start = 69, length = 15)

Does it make more sense to move this script to sonic-utilities repo? If yes, you don't need to manually install.

Reply:
Moved it to sonic-utilities repo.